### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by JxJxJxMxNxS**

### CODE REVIEW

File: common/src/main/java/nearsoft/skytouch/common/config/RabbitMQConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
+
+@EnableRabbit
+@Configuration
+public class RabbitMQConfiguration {
+
+    @Value("${rabbit-names-config.products-exchange-name:productExchange}")
```
Comment (iuresti) [inline common/src/main/java/nearsoft/skytouch/common/config/RabbitMQConfiguration.java:28]: I recommend to try the approach with the @ PropertyConfiguration class

File: management/src/main/java/nearsoft/skytouch/management/controller/HomeController.java [hunk 2]
```diff
@@ -6,7 +6,7 @@
 import java.util.Map;
 
 @Controller
-class HomeController {
+public class HomeController {
```
Comment (iuresti) [inline management/src/main/java/nearsoft/skytouch/management/controller/HomeController.java:9]: Just curiosity .... is there a reason why you adde `public` here ? did something fail?

File: microservice/src/test/java/nearsoft/skytouch/microservice/repository/ProductRepositoryTest.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+import javax.persistence.PersistenceContext;
+import javax.persistence.StoredProcedureQuery;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class ProductRepositoryTest {
```
Comment (iuresti) [inline microservice/src/test/java/nearsoft/skytouch/microservice/repository/ProductRepositoryTest.java:23]: I strongly recommend to change the test to validate the execution of the actual store procedures

File: microservice/src/main/java/nearsoft/skytouch/microservice/repository/impl/ProductRepositoryImpl.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+import javax.persistence.EntityManager;
+import javax.persistence.ParameterMode;
+import javax.persistence.PersistenceContext;
+import javax.persistence.StoredProcedureQuery;
+import javax.transaction.Transactional;
+import java.util.LinkedList;
+import java.util.List;
+
+@Repository
+public class ProductRepositoryImpl implements ProductRepository {
```
Comment (iuresti) [inline microservice/src/main/java/nearsoft/skytouch/microservice/repository/impl/ProductRepositoryImpl.java:17]: I'd like you to compare this approach with the followed by your team mates (Dilan and Sergio)

**Review by iuresti — APPROVED** [review]

File: common/src/main/java/nearsoft/skytouch/common/config/RabbitMQConfiguration.java (hunk 1 above)
Comment (JxJxJxMxNxS) [reply common/src/main/java/nearsoft/skytouch/common/config/RabbitMQConfiguration.java:28]: Yes i will do it, thanks

File: management/src/main/java/nearsoft/skytouch/management/controller/HomeController.java (hunk 2 above)
Comment (JxJxJxMxNxS) [reply management/src/main/java/nearsoft/skytouch/management/controller/HomeController.java:9]: I was trying to make an instance of the HomeController to make some testing and it caused some errors maybe the test was not in the same package i will add the HomeController tests

File: microservice/src/test/java/nearsoft/skytouch/microservice/repository/ProductRepositoryTest.java (hunk 3 above)
Comment (JxJxJxMxNxS) [reply microservice/src/test/java/nearsoft/skytouch/microservice/repository/ProductRepositoryTest.java:23]: Ok i will test it with a testing database thanks

File: microservice/src/main/java/nearsoft/skytouch/microservice/repository/impl/ProductRepositoryImpl.java (hunk 4 above)
Comment (JxJxJxMxNxS) [reply microservice/src/main/java/nearsoft/skytouch/microservice/repository/impl/ProductRepositoryImpl.java:17]: Ok i will ckeck their repos thanks

File: management/src/main/java/nearsoft/skytouch/management/controller/HomeController.java (hunk 2 above)
Comment (iuresti) [reply management/src/main/java/nearsoft/skytouch/management/controller/HomeController.java:9]: good, thanks

**Pull request merged**